book/src/config.md

```diff
@@ -170,6 +170,17 @@ Specifies the exact version which should be exempted.
 
 Specifies the criteria covered by the exemption.
 
+#### `dependency-criteria`
+
+Allows overriding the criteria requirements for dependencies on a per-dependency
+basis.  Allows overriding the above values on a per-dependency basis. Similar in
+format to the [equivalent field](audit-entries.md#dependency-criteria) in audit
+entries.
+
+This serves the same purposes as the field on audit entries, allowing the
+exemption to relax or strengthen the requirements which it places on
+dependencies when it is used.
```

Comment:
Maybe get more specific about the use-case: you might not have fully audited some crate but you can at least tell that a subtree can be mostly ignored.